src/dataprep/preprocess_utils.py

```diff
@@ -0,0 +1,118 @@
+import re
+import pandas as pd
+import numpy as np
+from typing import List, Tuple
+from sklearn.preprocessing import OrdinalEncoder, OneHotEncoder
+from sklearn.linear_model import LinearRegression
+from sklearn.experimental import enable_iterative_imputer  # IterativeImputer doesn't work without this import
+from sklearn.impute import IterativeImputer
+
+MISSINGNESS_PCT = 0.005
+
+
+def drop_small_missing_data(df: pd.DataFrame, features: List[str]) -> pd.DataFrame:
+    """
+    This method goes over the features with missing values that have very little rows with missing values and will
+    drop them.
+    E.g. dropping rows for features with less than 0.5% of missing values
+    """
```

Comment:
Make sure to use the Google Docstring documentation style.
```
    """Function description

    Args:
        param1: description
        param2: description

    Returns:
        return val1: description
        return val2: description
    """
```

Note that this is relevant to all the functions in this file and also to other files